visualization/app/codeCharta/ui/mapTreeView/mapTreeView.level.component.spec.ts

```diff
@@ -276,4 +292,45 @@ describe("MapTreeViewLevelController", () => {
 			expect(mapTreeViewLevelController["_viewModel"].collapsed).toBeFalsy()
 		})
 	})
+	describe("getNodeUnary", () => {
+		it("should return the unary of the current node", () => {
+			mapTreeViewLevelController["node"] = VALID_NODE_WITH_METRICS
+
+			const nodeUnary: number = mapTreeViewLevelController.getNodeUnary()
```

Comment:
No need to specify a type here, because the type of this is quite obvious and can be checked by your IDE hovering over the function.

Reply:
agreed